pyart/io/nexrad_level2.py

```diff
@@ -176,9 +178,15 @@ def __init__(self, filename):
 
         # pull out the vcp record
         msg_5 = [r for r in self._records if r['header']['type'] == 5]
+
+        wstring1 = 'No MSG5 detected. Setting to meaningless data \n'
+        wstring2 = 'Rethink your life choices and be ready for errors'
+
         if len(msg_5):
             self.vcp = msg_5[0]
         else:
+            # There is no VCP Data.. This is uber dodgy
+            warnings.warn(wstring1+wstring2)
```

Comment:
I would put

```
warnings.warn("No MSG5 detected. Setting to meaningless data. "
                        Rethink your life choices and be read for errors.")

Reply:
So no comma?

Reply:
No, the two statements are still printed together.